spec/classes/foreman_proxy__plugin__dhcp__infoblox_spec.rb

```diff
@@ -40,6 +40,8 @@
             :record_type  => 'host',
             :dns_view     => 'non-default',
             :network_view => 'another-non-default',
+            ':wait_after_restart: 99',
+            ":options: '[]'",          ])
```

Comment:
Syntax error here. Probably the same on line 57

Reply:
Sorry, fixed. It's just all red locally and I don't know why.